### PR TITLE
In ApplicationForm.tsx, update progress bar based on questions answered

### PR DESCRIPTION
**Pull request by vanessa-hu**
- made progressBarFill() function that calculates percent amt of questions answered based on total number of questions, to be used in the progress-bar divs in the render()
- rounds to whole number
- backtracks if input is removed
- also display percent number, can remove if needed
- realized that page numbers are just 0 out of 0? not sure if that is a bug or I just didn't pull since starting this branch

image

### CODE REVIEW

File: src/components/PDF/ApplicationForm.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ interface State {
 
 // Source: https://stackoverflow.com/questions/4998908/convert-data-uri-to-file-then-append-to-formdata
 function dataURLtoBlob(dataurl) {
-  const arr = dataurl.split(','); const mime = arr[0].match(/:(.*?);/)[1];
+  const arr = dataurl.split(','); // const mime = arr[0].match(/:(.*?);/)[1];
```
Comment (crchong1) [inline src/components/PDF/ApplicationForm.tsx:33]: can we remove this commented out code?

File: src/components/PDF/ApplicationForm.tsx [hunk 2]
```diff
@@ -182,6 +182,16 @@ class ApplicationForm extends Component<Props, State> {
     }
   }
 
+  progressBarFill() {
```
Comment (crchong1) [inline src/components/PDF/ApplicationForm.tsx:185]: arrow function (so do progressBarFill = () => {}) and also add a type declaration. This returns an int right? so lets define the return as an int

**Review by crchong1 — CHANGES_REQUESTED** [review]
just a couple small comments!

**Comment by vanessa-hu** [comment]
Done! Also didn't realize VSCode had in-house PR/Issue manager... so cool :O

**Review by crchong1 — APPROVED** [review]

**Pull request merged**